src/libraries/System.Private.Runtime.InteropServices.JavaScript/tests/System/Runtime/InteropServices/JavaScript/EnumMarshalTests.cs

```diff
@@ -0,0 +1,94 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.InteropServices.JavaScript;
+using System.Collections.Generic;
+using Xunit;
+
+namespace System.Runtime.InteropServices.JavaScript.Tests
+{
+    public static class EnumMarshalTests
+    {
+        public enum RequestCache
+        {
+            [EnumExport(EnumValue = ConvertEnum.Default)]
+            Default = -1,
+            [EnumExport("no-store")]
+            NoStore,
+            [EnumExport(EnumValue = ConvertEnum.ToUpper)]
+            Reload,
+            [EnumExport(EnumValue = ConvertEnum.ToLower)]
+            NoCache,
+            [EnumExport("force-cache")]
+            ForceCache,
+            OnlyIfCached = -3636,
+        }
+
+        public enum StyleColor : uint
+        {
+            RedNoExport,
+            [EnumExport(EnumValue = ConvertEnum.Default)]
+            Red = 0xff0000ff,
+            [EnumExport(EnumValue = ConvertEnum.Numeric)]
+            GreenNumeric = 0x00ff00ff,
+            [EnumExport("red", EnumValue = ConvertEnum.ToUpper)]
+            RedUpperCase,
+            [EnumExport("RED", EnumValue = ConvertEnum.ToLower)]
+            RedLowerCase,
+            [EnumExport("#ff0000")]
+            RedHex
+        }
+
+        [Fact]
+        public static void MarshalRequestEnums()
```

Comment:
nit: Should this be renamed to `StringFromJSToManagedEnum()`? or `JSStringToManaged`, so this sounds like `JSString` is some type being used here.

Reply:
Not sure I understand why this is needed or a nit.  Could you explain this more as to why it is needed.

Reply:
Just as a more descriptive name for the test. `MarshalStringFromJSToManagedEnum`?

Reply:
Gotcha.  Addressed in recent commits